debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/OptInfo.java

```diff
@@ -266,4 +267,30 @@ public static J9UTF8Pointer getPermittedSubclassNameAtIndex(J9ROMClassPointer ro
 		return J9UTF8Pointer.NULL;
 	}
 
+	public static int getPreloadClassCount(J9ROMClassPointer romClass) throws CorruptDataException {
+		U32Pointer preloadClassInfoPointer = getPreloadClassInfoPointer(romClass);
+		if (preloadClassInfoPointer.notNull()) {
+			return preloadClassInfoPointer.at(0).intValue();
+		}
+		return 0;
+	}
+
+	private static U32Pointer getPreloadClassInfoPointer(J9ROMClassPointer romClass) throws CorruptDataException {
+		SelfRelativePointer srpPtr = getSRPPtr(J9ROMClassHelper.optionalInfo(romClass), romClass.optionalFlags(),
+			J9_ROMCLASS_OPTINFO_PRELOAD_ATTRIBUTE);
+		if (srpPtr.notNull()) {
+			return U32Pointer.cast(srpPtr.get());
+		}
+		return U32Pointer.NULL;
+	}
+
+	public static J9UTF8Pointer getPreloadClassNameAtIndex(J9ROMClassPointer romClass, int index) throws CorruptDataException {
+		U32Pointer preloadClassInfoPointer = getPreloadClassInfoPointer(romClass);
+		if (preloadClassInfoPointer.notNull()) {
+			/* extra 1 is to move past preload class count. */
```

Comment:
nit: Please fix this comment. If it's a sentence, it should start with a capital, if not, it doesn't end with a period.